openquakeplatform_ipt/static/ipt/js/app_web.js

```diff
@@ -0,0 +1,298 @@
+/**
+ *
+ * @source: hybridge (chrome) ipt.js
+ * @author: Matteo Nastasi <nastasi@alternativeoutout.it>
+ * @link: https://github.com/nastasi/hybridge
+ *
+ * @licstart  The following is the entire license notice for the
+ *  JavaScript code in this page.
+ *
+ * Copyright (C) 2018 Matteo Nastasi
+ *
+ *
+ * The JavaScript code in this page is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU
+ * General Public License (GNU GPL) as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.  The code is distributed WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.
+ *
+ * As additional permission under GNU GPL version 3 section 7, you
+ * may distribute non-source (e.g., minimized or compacted) forms of
+ * that code without the copy of the GNU GPL normally required by
+ * section 4, provided you include this license notice and a URL
+ * through which recipients can access the Corresponding Source.
+ *
+ * @licend  The above is the entire license notice
+ * for the JavaScript code in this page.
+ *
+ */
+
+function manage_error_cb(uuid, msg)
+{
+    if ('result' in msg) {
+        if ('reason' in msg.result) {
+            gem_ipt.error_msg(msg.result.reason);
+            return;
+        }
+    }
+    if (type(console) !== 'undefined') {
+        if ('log' in console) {
+            console.log('manage_error_cb');
+            console.log(msg);
+            gem_ipt.error_msg('A command executed by QGIS returned a malformed reply, take a look to  console.log for more informations');
```

Comment:
Please remove the double space before console.log, and keep information without a final 's'